File: Python PowerPoint Examples/ShapesApiDemo/setSlideShapeParagraph.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.shapes_api.ShapesApi(configuration)

fileName = "SetSlideShapeParagraph.pptx"
password = ""
storage = ""
inputFolder = "input"
slide_index = 0
shape_index = "0"
paragraph_index = 0
source = spirecloudpowerpoint.models.Paragraph()
source.text_alignment = "Baseline"
source.alignment = "Center"
source.margin_left = 50
source.bullet_type = "None"
result =api.set_slide_shape_paragraph(fileName, slide_index, shape_index, paragraph_index,source=source, password=password, storage=storage,folder=inputFolder)

File: JavaScript PowerPoint Examples/ShapesApiDemo/setSlideShapeParagraph.js
(function (){
    var appId = "your ID";
    var appKey = "your Key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudPowerpoint = require('../../src/index');
    var apiClient = new SpirecloudPowerpoint.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudPowerpoint.ShapesApi(apiClient);
    
    var name = 'SetSlideShapeParagraph.pptx';
    var folder = 'input';
    var slideIndex = 0;
    var shapeIndex = 0;
    var paragraphIndex = 0;

    var opts = {};
    opts.source = new SpirecloudPowerpoint.Paragraph();
    opts.source.MarginLeft = 50;
    opts.source.TextAlignment = 'Baseline';
    opts.source.Alignment = 'Center';
    opts.folder = folder;
    
    var response = instance.setSlideShapeParagraph(name, slideIndex, shapeIndex, paragraphIndex, opts,
     function (error, data, response) {
        if (error) throw error;
     });
})();